content/applications/finance/accounting/taxes/vat_verification.rst

```diff
@@ -17,7 +17,7 @@ Odoo's interface.
 VIES VAT number verification
 ============================
 
-To activate this feature, go to :menuselection:`Accounting --> Configuration --> Settings`. In the
+To activate this feature, install the module base_vat and then go to :menuselection:`Accounting --> Configuration --> Settings`. In the
```

Comment:
```suggestion
To activate this feature, install the module :guilabel: `VAT Number Validation (base_vat)` from **Apps** and then go to :menuselection:`Accounting --> Configuration --> Settings`. In the
```

Reply:
I believe this is what the user would see, not the technical name. I'd push it myself but I can't access your branch :)